File: beginner/pagina11/2968/2968.js
const fs = require('fs')
const input = fs.readFileSync("/dev/stdin", "utf8").split(" ").map(Number)
const [voltas, placas] = input

const total_placas = voltas * placas;
let output = []

for(let i = 10; i <= 90; i += 10){
    let qnt_placas_por_porcentagem = Math.ceil((total_placas * i) / 100)
    output.push(qnt_placas_por_porcentagem)
}

console.log(output.join(" ").trim())

File: beginner/pagina11/2968/2968.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);

        String[] input = in.nextLine().split(" ");
        int voltas = Integer.parseInt(input[0]);
        int placas = Integer.parseInt(input[1]);

        double total_placas = (double) voltas * placas;

        StringBuilder output = new StringBuilder();
        for(int i = 10 ; i <= 90; i+= 10){
            int qnt_placas = (int) Math.ceil((total_placas * i) / 100.0);
            if(output.length() > 0){
                output.append(" ");
            }
            output.append(qnt_placas);
        }

        System.out.println(output);
        in.close();
    }
}